<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XenonVM</class>
 <widget class="QMainWindow" name="XenonVM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Xenon Virtual Machine</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Xenon.png</normaloff>../Xenon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(9, 29, 32);
font-family: &quot;JetBrains Mono&quot;, &quot;Consolas&quot;;
font-size: 14px;
font-weight: 600;
color: white;
border-radius: 10px;
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="load_file_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton {
    background-color: rgb(92, 102, 114);
    border-radius: 10px;
    padding: 5px;
}
QPushButton:hover {
    background-color: rgb(108, 111, 127);
}
     </string>
    </property>
    <property name="text">
     <string>Load file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="process_button">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>40</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton {
    background-color: rgb(92, 102, 114);
    border-radius: 10px;
    padding: 5px;
}
QPushButton:hover {
    background-color: rgb(108, 111, 127);
}
     </string>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="file_text">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>100</y>
      <width>350</width>
      <height>420</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(13, 37, 41);
color: rgb(221, 184, 126);
padding: 6px;
font-family: &quot;JetBrains Mono&quot;, &quot;Consolas&quot;;
border-radius: 10px;
     </string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'JetBrains Mono,Consolas'; font-size:14px; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="tabStopDistance">
     <double>31.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="step_button">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>40</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton {
    background-color: rgb(92, 102, 114);
    border-radius: 10px;
    padding: 5px;
}
QPushButton:hover {
    background-color: rgb(108, 111, 127);
}
     </string>
    </property>
    <property name="text">
     <string>Step</string>
    </property>
   </widget>
   <widget class="QPushButton" name="export_button">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton {
    background-color: rgb(92, 102, 114);
    border-radius: 10px;
    padding: 5px;
}
QPushButton:hover {
    background-color: rgb(108, 111, 127);
}
     </string>
    </property>
    <property name="text">
     <string>Export</string>
    </property>
   </widget>
   <widget class="QLabel" name="a_reg">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 170, 0)</string>
    </property>
    <property name="text">
     <string>A: 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="d_reg">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(100, 200, 255)</string>
    </property>
    <property name="text">
     <string>D: 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="memory">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white</string>
    </property>
    <property name="text">
     <string>M: 0</string>
    </property>
   </widget>
   <widget class="QSlider" name="clock_speed">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="clock_speed_label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>25</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Clock speed: Instant</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
